src/Illuminate/Database/Query/Builder.php

```diff
@@ -1299,6 +1299,32 @@ public function orWhereYear($column, $operator, $value = null)
         return $this->addDateBasedWhere('Year', $column, $operator, $value, 'or');
     }
 
+    /**
+     * Add an "where today" statement in the query.
+     *
+     * @param  string $column
+     * @param \DateTimeZone|null $timeZone
+     * @return \Illuminate\Database\Query\Builder|static
+     * @throws \Exception
+     */
+    public function whereToday($column, ?\DateTimeZone $timeZone = null)
+    {
+        return $this->whereDate($column, '=', new \DateTime('now', $timeZone));
```

Comment:
Just FTR (not that it matters anymore), it should have been `\Illuminate\Support\Carbon` => this is the default the framework uses (it's defined in the `illuminate/support` package which other, framework independent ones, usually depend one).